modules/jaeger-features.adoc

```diff
@@ -1,10 +1,10 @@
 ////
-This CONCEPT module included in the following assemblies:
+This module included in the following assemblies:
```

Comment:
Refer to this guideline: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata

Suggestion: Even if it is just comment out, it is good to follow what is prescribed as per the guideline as it ensures consistency throughout the files. (Global comment)